index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Amr Ahmed - Portfolio</title>
+</head>
+<body>
+    <img src="/images/top_circles.png" alt="Background image circles white">
+    <!-- <img src="/images/portfoliobg.png" alt="Background image circles white"> -->
+    <div class="wrapper">
+        <div class="header">
+            <span>Amr Ahmed</span> <div class="hamburger-menu"><i class="fas fa-bars"></i></div>
+        </div>
+        <div class="headline-section">
+            <span class="myName">
+                Amr Ahmed
+            </span>
+            <span class="role">Software Developer</span>
```

Comment:
- [ ] For the sub-title you can use any of the `hx` tags so long as the hierarchy is below the `h1`.